src/main/resources/db/changelog/changeset/0046_remove_feeback_related_columns/0046_changeSet.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+  <changeSet author="tkuzynow" id="migrateUserNotificationSettings">
```

Comment:
please change id - I think it should really be unique
the author can also adjusted, even it's not really important ()